4.1 release (#1047)
watson-developer-cloud/swift-sdk: README.md

## Installation

The IBM Watson Swift SDK can be installed with [Swift Package Manager](https://swift.org/package-manager/), [Cocoapods](http://cocoapods.org/), or [Carthage](https://github.com/Carthage/Carthage).

### Swift Package Manager

> New in version 4.1, the Watson Developer Cloud Swift SDK now supports all services through Swift Package Manager.

Reviewer: This references version `4.1`, but so far the highest commit level is a `fix`. This means that the semantic release is likely to resolve to version `4.0.2`.

Author: My bad I thought a fix resolved to a minor version update

On the XCode menu bar at the top of your screen click `File -> Swift Packages -> Add Package Dependencies`, follow the prompts by pasting the github url `https://github.com/watson-developer-cloud/swift-sdk` and using the most recent major version if appropriate. Make sure to only click on the services you plan on using otherwise you may face long build times

To import a service into your project:
```swift

> import AssitantV2

Reviewer: ```suggestion
import AssistantV2
```

import DiscoveryV2
.
.
.
```

> (**Speech To Text and Text To Speech only**)

Reviewer: ```suggestion
**(Speech To Text and Text To Speech only)**
```

> The use of special libraries by these services requires extra steps. 

Reviewer: ```suggestion
The use of the `libogg` and `opus` libraries by these services requires extra steps to be taken.
```

2. Install `libogg` and `opus`
    ```bash
    $ brew install libogg opus
    ```
3. Packaged dynamic libraries must be removed according to current library versions.

    libogg version as of writing: 1.3.4. 
    
    opus version as of writing: 1.3.1

    ```bash
    $ rm -f /usr/local/Cellar/libogg/1.3.4/lib/*.dylib
    ```
    ```bash
    $ rm -f /usr/local/Cellar/opus/1.3.1/lib/*.dylib
    ```
4. The static libraries installed must be replaced with libraries compiled for multiple architectures. These libraries can be downloaded from the github repo for libogg [here](https://github.com/watson-developer-cloud/swift-sdk/blob/master/Sources/SupportingFiles/Dependencies/Libraries/libogg.a) and opus [here](https://github.com/watson-developer-cloud/swift-sdk/blob/master/Sources/SupportingFiles/Dependencies/Libraries/libopus.a)

5. Replace the currently installed `libogg` and `libopus` libraries
    ```bash
    rm /usr/local/Cellar/libogg/1.3.4/lib/libogg.a && cp ~/Downloads/libogg.a /usr/local/Cellar/libogg/1.3.4/lib
    ```
    ```bash
    rm /usr/local/Cellar/libogg/1.3.1/lib/libopus.a && cp ~/Downloads/libopus.a /usr/local/Cellar/opus/1.3.1/lib
    ```

> 6. You're ready to go!

Reviewer 2: 🚀

use_frameworks!

target 'MyApp' do

>     pod 'IBMWatsonAssistantV1', '~> 4.0'

Reviewer: These version numbers are updated by the release automation and shouldn't be hand edited.

If your project does not have a Cartfile yet, use the `touch Cartfile` command in the root directory of your project. To install the IBM Watson Swift SDK using Carthage, add the following to your Cartfile.

```

> github "watson-developer-cloud/swift-sdk" ~> 4.0

Reviewer: Same as with the pod installation instructions above, this version number shouldn't be hand edited.
